credsweeper/__init__.py

```diff
@@ -8,8 +6,6 @@
 from credsweeper.ml_model.ml_validator import MlValidator
 from credsweeper.validations.apply_validation import ApplyValidation
 
-os.environ["TF_CPP_MIN_LOG_LEVEL"] = "2"
-
```

Comment:
As I know this line is needed to avoid warning log..
Could you explain why this line should be removed?

Reply:
The environment used to manage log in tensorflow package. Since we use onnxruntime - it is extra.

Reply:
Oh.. That's right. Thank you for remove useless lines.